src/routes/jira/connect/enterprise/jira-connect-enterprise-post.test.ts

```diff
@@ -136,4 +140,34 @@ describe("POST /jira/connect/enterprise", () => {
 			}]
 		});
 	});
+
+	it("POST Jira Connect Enterprise - invalid status code still return success when relax ff on", async () => {
+
+		when(booleanFlag)
+			.calledWith(BooleanFlags.RELAX_GHE_URLS_CHECK, expect.anything())
+			.mockResolvedValue(true);
+
+		const response = mockResponse();
+		gheNock.get("/").reply(500);
+		await JiraConnectEnterprisePost(mockRequest(gheUrl), response);
+		expect(response.status).toHaveBeenCalledWith(200);
+		expect(response.send).toHaveBeenCalledWith({ success: true, appExists: false });
+	});
+
+	it("POST Jira Connect Enterprise - network error code will fail when relax ff on", async () => {
+
+		when(booleanFlag)
+			.calledWith(BooleanFlags.RELAX_GHE_URLS_CHECK, expect.anything())
+			.mockResolvedValue(true);
+
+		const response = mockResponse();
+		await JiraConnectEnterprisePost(mockRequest(gheUrl), response);
+		expect(response.status).toHaveBeenCalledWith(200);
+		expect(response.send).toHaveBeenCalledWith({
+			success: false, errors: [{
+				code: "GHE_ERROR_CANNOT_CONNECT",
+				reason: expect.stringMatching(/ENOTFOUND|EAI_AGAIN/)
```

Comment:
`/ENOTFOUND|EAI_AGAIN/` is to support running test on both locally and on github action.